internal/controller/metrics_test.go

```
@@ -0,0 +1,101 @@
+		secretName = "missing-secret"
+	)
+
+	grant := &gatewayv1beta1.ReferenceGrant{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "grant-metric-test",
+			Namespace: secretNS,
+		},
+		Spec: gatewayv1beta1.ReferenceGrantSpec{
+			From: []gatewayv1beta1.ReferenceGrantFrom{{
+				Group:     gatewayv1beta1.Group(batchv1alpha1.GroupVersion.Group),
+				Kind:      "LLMBatchGateway",
```

golangci-lint (2.12.2)

[error] string `LLMBatchGateway` has 6 occurrences, but such constant `llmBatchGatewayKind` already exists

(goconst)